### PR TITLE
Exposing fine tune functionality when training a model

### PR DESCRIPTION
**Pull request by lperryp**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Exposing fine tune functionality when training a model

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml [hunk 1]
```diff
@@ -151,10 +151,10 @@ args:
 comment: Create a phishing classifier using machine learning technique, based on email
   content.
 commonfields:
-  id: DBotBuildPhishingClassifier
```
Comment (erezh31) [inline Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml:134]: i think you mistake with the ID please change

File: Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py [hunk 2]
```diff
@@ -351,6 +374,11 @@ def main():
     keyword_min_score = float(demisto.args()['keywordMinScore'])
     return_predictions_on_test_set = demisto.args().get('returnPredictionsOnTestSet', 'false') == 'true'
     original_text_fields = demisto.args().get('originalTextFields', '')
+    algorithm = demisto.args().get('trainingAlgorithm', AUTO_TRAINING_ALGO)
+    # FASTTEXT_TRAINING_ALGO and FROM_SCRATCH_TRAINING_ALGO are equivalent, replacement is done because ml_lib
```
Comment (erezh31) [inline Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py:378]: remove hidden code

**Review by erezh31 — CHANGES_REQUESTED** [review]

File: Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml (hunk 1 above)
Comment (lperryp) [reply Packs/Base/Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.yml]: your'e right, will fix this, thanks

File: Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py (hunk 2 above)
Comment (lperryp) [reply Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py:378]: It's a comment, wanted to clarify why in the UI it's presented as "from_scratch" but in the script it's changed to "fasttext"

**Comment by ShirleyDenkberg** [comment]
@erezh31 Doc review completed.

**Review by erezh31 — APPROVED** [review]

**Pull request merged**